lib/oli_web/templates/layout/delivery.html.heex

```diff
@@ -112,8 +112,11 @@
     >
     </script>
 
-    <script src="https://polyfill.io/v3/polyfill.min.js?features=es6">
-    </script>
+    <% Map.get(assigns, :vr_agent_active) |> IO.inspect(label: "---A2") %>
```

Comment:
Remove this line as it seems like it is here just for debugging

Reply:
Thanks for catching this.